README.md

```diff
@@ -32,6 +32,8 @@ General guidelines for Hellbender developers
 
 * Untested code is considered **non-existent** and thus is subject to removal at any time (exception is main methods, or super corner conditions, or `toString()` code).
 
+* Each tool should have at least one good end-to-end integration test with a check for expected output, plus high-quality unit tests for all non-trivial utility methods/classes used by the tool. Although we have no specific coverage target, coverage should be extensive enough that if tests pass, the tool is guaranteed to be in a usable state.
```

Comment:
Guys, @dekling has been going through the Picard tools defining minimal command lines for the basic use cases. If it helps, we can ask him to post the relevant info in each tool's test ticket so you guys can write the integration test at least. The second round of this project will be evaluating that the output is actually correct. If we find any problems at that stage we can revisit the corresponding tickets at that time.